usage-stats-handler/app.js

```diff
@@ -79,16 +79,20 @@ server.post('/grafana-usage-report', function (req, res, next) {
 
   // elastic.saveReport(report);
 
-  // group versions like 8_2_0-33922pre as 8_2_0-pre
-  const cleanVersion = report.version.replace(/-[a-f0-9]+pre$/, '-pre');
+  // group versions like 
+  // 8_2_0-33922pre as 8_2_0
+  // 8_2_0-12341343 as 8_2_0
+  // This reduces the cardinality of the versions stored in Graphite
+  // to look at more precis versions you need to look at big query stats
+  
+  const cleanVersion = report.version.replace(/-.+/, '');
```

Comment:
@DanCech By only tracking major version regardless if its beta, pre release etc we should drive down metrics a lot. For public metrics that should be enough.